docs/tool_interface.rst

```diff
@@ -157,10 +157,15 @@ View-components definition
 
```

Comment:
I already rebased locally and add my changes, do not push more commits, they will be overwritten